woocommerce/payment-gateway/admin/class-sv-wc-payment-gateway-admin-order.php

```diff
@@ -81,31 +78,54 @@ public function __construct( SV_WC_Payment_Gateway_Plugin $plugin ) {
 	 */
 	public function enqueue_scripts( $hook_suffix ) {
 
-		if ( 'shop_order' === get_post_type() && 'post.php' === $hook_suffix ) {
+		// Order screen assets
+		if ( 'shop_order' === get_post_type() ) {
 
-			wp_enqueue_script( 'sv-wc-payment-gateway-admin-order', $this->get_plugin()->get_payment_gateway_framework_assets_url() . '/js/admin/sv-wc-payment-gateway-admin-order.min.js', array( 'jquery' ), SV_WC_Plugin::VERSION, true );
+			// Edit Order screen assets
+			if ( 'post.php' === $hook_suffix ) {
```

Comment:
this condition can be merged in the previous statement as it was before, no?

Reply:
You can write each concatenated `&&` condition in a new line inside a single if statement - it makes clear from the beginning there are multiple requirements for a condition. If this is moved below I might assume there's then an if/else branch, at a glance.

Reply:
I'm not a fan of the multi-line if statements generally, and in this case I'm preferring to keep each statement separate since these are conditions for when certain scripts are enqueued.

The first checking for the general order post type, then specifically the edit screen, etc... which makes it easy to add other enqueue methods that are meant for different screens without refactoring the conditions.